### PR TITLE
TestNG adaptor docs (fixes #22)

### LINKED ISSUE #22: [TestNG] Latest adaptor updates

### PR DESCRIPTION
**Pull request by sskorol**
- updated `TestNG` adaptor docs according to latest allure 2.0 updates;
 - fixed typos in root `index.html`;

### CODE REVIEW

File: docs/2.0/integration/testng.adoc [hunk 1]
```diff
@@ -1,29 +1,41 @@
-= TestNG
+=== TestNG
```
Comment (eroshenkoam) [inline docs/2.0/integration/testng.adoc]: Decrease leveloffset, please)

File: docs/2.0/integration/testng.adoc [hunk 2]
```diff
@@ -1,29 +1,41 @@
...
 An Allure report can be generated for any TestNG test. In order to get test results, you need to:
 
-* Add AllureTestListener to TestNG settings.
-* Add AspectJ Weaver dependency and its properties.
+* Download and uzip latest **allure-command-line**: https://bintray.com/qameta/generic/download_file?file_path=io%2Fqameta%2Fallure%2Fallure%2F2.0-BETA5%2Fallure-2.0-BETA5.zip
```
Comment (eroshenkoam) [inline docs/2.0/integration/testng.adoc]: https://dl.bintray.com/qameta/generic/io/qameta/allure/allure/2.0-BETA5/allure-2.0-BETA5.zip

File: docs/2.0/integration/testng.adoc [hunk 3]
```diff
@@ -66,205 +104,119 @@ Then run the build as usual:
...
-</target>
-----
-
-Don't forget to specify **build.test-compile** and **build.compile** properties pointing to your software source code and test *.class files, respectively. Then launch the build as usual:
-
+Allure results will appear in **target/allure-results** folder. To generate html report and automatically open it in a web browser, run the following command:
 [source, bash]
 ----
-$ ant test
+$ ./pathToCommandLine/allure-2.0-BETA5/bin/allure serve /projectRoot/target/allure-results
```
Comment (eroshenkoam) [inline docs/2.0/integration/testng.adoc]: allure serve target/allure-results

File: docs/2.0/integration/testng.adoc [hunk 4]
```diff
@@ -66,205 +104,119 @@ Then run the build as usual:
...
-    )
-    public void searchTest() throws Exception {
-        ...
-    }
 
-}
+Allure results will appear in **build/allure-results** folder. To generate html report and automatically open it in a web browser, run the following command:
+[source, bash]
+----
+$ ./pathToCommandLine/allure-2.0-BETA5/bin/allure serve /projectRoot/build/allure-results
```
Comment (eroshenkoam) [inline docs/2.0/integration/testng.adoc]: allure serve build/allure-results

File: docs/2.0/integration/testng.adoc (hunk 1 above)
Comment (sskorol) [inline docs/2.0/integration/testng.adoc]: @eroshenkoam do you mean reverting to a single `=`?.. if yes, for all the titles or only for this one?

File: docs/2.0/integration/testng.adoc (hunk 2 above)
Comment (ehborisov) [inline docs/2.0/integration/testng.adoc]: you can wrap the link, https://dl.bintray.com/qameta/generic/io/qameta/allure/allure/2.0-BETA5/allure-2.0-BETA5.zip [**allure-commandline**]

File: docs/2.0/integration/testng.adoc (hunk 1 above)
Comment (eroshenkoam) [inline docs/2.0/integration/testng.adoc]: For all titles replace "===" -> "=", "====" -> "==", etc

Comment (sskorol) [inline docs/2.0/integration/testng.adoc]: @eroshenkoam ok, will fix it

**Comment by sskorol** [comment]
@eroshenkoam @baev fixed comments

**Pull request merged**